File: Warmup1_PYTHON/front_back.py
'''
PROMPT
Given a string, return a new string where the
first and last chars have been exchanged.
'''

def front_back(str):
  if len(str) <= 1:
    return str

  center = str[1:-1]

  return str[-1:] + center + str[0]

'''
You can return a substring of unknown length that
excludes the last character by typing str[x:-1]. This
can also be used in conjunction with a lack of parameter,
like in the return statement on Line 13.
'''

File: Warmup1_JAVA/frontBack.java
/*
PROMPT
Given a string, return a new string where the first
and last chars have been exchanged.
*/

public String frontBack(String str) {
  int len = str.length();
  if (len > 1) {
    return str.charAt(len-1) + str.substring(1, len-1) + str.charAt(0);
  }
  return str;
}

/*
To switch the first and last characters, you can also create variables
and call those instead. I just returned a string by returning
the last character + middle substring + first character.
*/
